mne/stats/regression.py

```diff
@@ -134,3 +143,208 @@ def _fit_lm(data, design_matrix, names):
         mlog10_p_val[predictor] = -np.log10(p_val[predictor])
 
     return beta, stderr, t_val, p_val, mlog10_p_val
+
+
+def linear_regression_raw(raw, events, event_id=None,
+                          tmin=-.1, tmax=1,
```

Comment:
not sure we need `tmin` and `tmax` since it's pretty easy to crop raw instances, WDYT?